app/content/content.ts

```diff
@@ -167,7 +167,7 @@ class Intercept {
         }
       })();`;
     let script = document.createElement("script");
-    script.defer = true;
+    script.defer = false;
```

Comment:
I was having one concern about this change. Does making make everypage load `slower` since this script has be loaded and executed before rendering the page iteself ?

Reply:
can you test this out and come up with some stats?